pkgs/development/python-modules/distributed/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pytest
+, click
+, cloudpickle
+, dask
+, msgpack
+, psutil
+, six
+, sortedcontainers
+, tblib
+, toolz
+, tornado
+, zict
+, pyyaml
+, pythonOlder
+, futures
+, singledispatch
+}:
+
+buildPythonPackage rec {
+  pname = "distributed";
+  version = "1.22.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "eefdd511912a001077bf1e00e2c3989dcfa853d68d14a8cd5b688c6175389e3a";
+  };
+
+  checkInputs = [ pytest ];
+  propagatedBuildInputs = [
+      click cloudpickle dask msgpack psutil six
+      sortedcontainers tblib toolz tornado zict pyyaml
+  ] ++ lib.optional (pythonOlder "3.2") futures
+    ++ lib.optional (pythonOlder "3.4") singledispatch;
+
+  checkPhase = ''
+    py.test distributed
+  '';
+
+  # py.test not picking up local config file, even when running
+  # manually: E ValueError: no option named '--runslow'
+  doCheck = false;
```

Comment:
Maybe you need to add pytest-repeat or pytest-faulthandler to checkInputs.

Reply:
See https://github.com/dask/distributed/blob/master/continuous_integration/travis/install.sh#L57